### PR TITLE
Set marker

### PR DESCRIPTION
**Pull request by UrsulaMontag**
This PR is for the markers on each entry-position. I also implemented Zustand to prevent prop-drilling. All changes in EntryList and index come from the implementation of state.
(see #14 )

### CODE REVIEW

File: src/components/marker/EntryMarker.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { Marker, Popup } from 'react-leaflet';
+import useStore from '../../lib/useStore.js';
+
+export default function EntryMarker() {
+	const entries = useStore(state => state.entries);
+	console.log(entries);
+	return (
```
Comment (irenepapaspyratos) [inline src/components/marker/EntryMarker.jsx:7]: Are you sure, you need the console.log here?

**Review by irenepapaspyratos — COMMENTED** [review]
Besides the small things, I would have made different, all looks fine! 🙃

**Review by irenepapaspyratos — APPROVED** [review]

File: src/components/marker/EntryMarker.jsx (hunk 1 above)
Comment (UrsulaMontag) [reply src/components/marker/EntryMarker.jsx:7]: No, thank you. I have overlooked it. It's cleared now

File: src/lib/useStore.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import create from 'zustand';
+
+const useStore = create(set => {
+	console.log(set);
+	return {
```
[the remark this answers is not in the document]
Comment (UrsulaMontag) [reply src/lib/useStore.js:5]: same

File: src/lib/useStore.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+		entries: [
+			{
+				id: 1,
```
[the remark this answers is not in the document]
Comment (UrsulaMontag) [reply src/lib/useStore.js:8]: These are just the testing-data. Later I have to fetch the database, and will replace the test-data in the useStore with the fetched data. At this time, this is the simple method I chose to keep it as simple as possible.

**Pull request merged**